### PR TITLE
fix: performance optimization

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [x] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

使用 `translate3d` 代替 `translateX`，可以调用GPU进行3D加速

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix: performance optimization |
| 🇨🇳 Chinese | 性能优化 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .github/workflows/ui-upload.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+name: 📷 UI Upload
+
+on:
+  workflow_run:
+    workflows: ["📷 UI"]
+    types:
+      - completed
+
+permissions:
+  contents: read
```
Comment (afc163) [inline .github/workflows/ui-upload.yml:11]: 不应该改这些

Comment (li-jia-nan) [reply .github/workflows/ui-upload.yml:11]: 我以为上游分支是next……同步了一下结果直接close了

Comment (afc163) [reply .github/workflows/ui-upload.yml:11]: 怎么同步的，感觉有问题。

Comment (li-jia-nan) [reply .github/workflows/ui-upload.yml:11]: C875D9FE-8AEF-4E80-AC16-5B90433072A8

看到上游分支有更新，就点了这个按钮，然后就自动close了
因为这个分支是从next分支创建的，所以上游分支默认就是next，不知道为啥会同步出问题

**Pull request closed without merging**